feature-libs/qualtrics/components/qualtrics-loader/config/default-qualtrics-config.ts

```diff
@@ -0,0 +1,5 @@
+import { QualtricsConfig } from './qualtrics-config';
+
+export const defaultQualtricsConfig: QualtricsConfig = {
+  qualtrics: {},
```

Comment:
In the `qualtrics-config.ts` we say that the `scriptSource` defaults to a particular path, but that path is not set here.
Let's either: 
- update the comment
- add the default path here

Reply:
added default path as stated in the comment